### ISSUE TITLE
can't use Iguanatex anymore since Microsoft blocked all macros a couple days ago

### ISSUE THREAD

**Issue by danguelo** [Jonathan-LeRoux/IguanaTex#28]
I wanted to work with my powerpoint presentation today and saw a message that said the macro was blocked by Microsoft because my file was not trusted, then I read that Microsoft blocked all macros in Office by default a couple days ago, supposedly I can unblock them by clicking right in the powerpoint file and mark the unlock option, but not really, nothing happened.

I then saw in the PP options that Iguanatex was disabled, whenever I try to enable it again a message appears saying it is a potential security risk. I deleted Iguanatex and tried to add it again and the same message appears. I also Enabled all macros in the TrustCenter in Powerpoint. So it seems Microsoft does not want me to use Iguanatex, or is there anything I'm doing wrong? (attached image is in spanish, sorry I'm not an english speaker).

This is where  Microsoft stated the Macro block https://docs.microsoft.com/DeployOffice/security/internet-macros-blocked

![pp](https://user-images.githubusercontent.com/103894956/163723579-831697e7-08a8-4442-b243-0ecd4f68ca44.png)

**Comment by LePingKYXK** [comment]
> I wanted to work with my powerpoint presentation today and saw a message that said the macro was blocked by Microsoft because my file was not trusted, then I read that Microsoft blocked all macros in Office by default a couple days ago, supposedly I can unblock them by clicking right in the powerpoint file and mark the unlock option, but not really, nothing happened.
> 
> I then saw in the PP options that Iguanatex was disabled, whenever I try to enable it again a message appears saying it is a potential security risk. I deleted Iguanatex and tried to add it again and the same message appears. I also Enabled all macros in the TrustCenter in Powerpoint. So it seems Microsoft does not want me to use Iguanatex, or is there anything I'm doing wrong? (attached image is in spanish, sorry I'm not an english speaker).
> 
> This is where Microsoft stated the Macro block https://docs.microsoft.com/DeployOffice/security/internet-macros-blocked
> 

Hi danguelo,

Here is my solution. I was using the IguanaTex of version1.57 in the Office 365. It was unable to work 2 days ago. 
- First step, I deleted it from the PowerPoint Add-in and then download the version 1.58.
- Second step, I ticked the `[x] Require macros to be signed by a trusted publisher` in the `Trust Center Settings`. 
    You can find it by clicking `File` --> `Options` --> `Trust Center` --> `Trust Center Settings` --> `Add-in` (I am sorry for attaching the screenshot in Chinese, but I labeled the order with numbers).
- Third step, I added the IguanaTex into the Add-in. 

I hope it will help.

Good luck and Best wishes from China.

Huan

![trust_center](https://user-images.githubusercontent.com/14786161/164894232-63c65d3d-6b83-4567-8fde-12140232c978.png)

**Comment by Jonathan-LeRoux** [comment]
Thanks for reporting this. Whether someone experiences this issue may depend on various things, such as their institution's security settings, how they downloaded the add-in, and where they saved it.

It sounds like the easiest solution, without requiring a signed version of the add-in, would be to either "unblock" the .ppam add-in file via its properties in Explorer, or save it in a [Trusted Location](https://docs.microsoft.com/en-us/DeployOffice/security/trusted-locations), as explained in [this part](https://docs.microsoft.com/en-us/DeployOffice/security/internet-macros-blocked#additional-information) of the Microsoft article that @danguelo pointed to. For example, if you save the .ppam file in `%appdata%\Microsoft\Addins` (i.e., `Users\user_name\Appdata\Roaming\Microsoft\Addins`), I think you should be able to use the add-in (unless your institution specifically prevents it).

**Comment by tgkolda** [comment]
Comments by @Jonathan-LeRoux worked for me. Thanks!

**Comment by Jonathan-LeRoux** [comment]
I'm closing this issue as there were no further comments from OP. Please reopen as needed.

**Comment by chibuzonnonyelu** [comment]
This worked. Thanks.

**Comment by TeaPearce** [comment]
these fixes didn't work for me. unclear if it's the exact same issue -- for me Windows Security blocks it.

![image](https://user-images.githubusercontent.com/25795223/234529567-32865e8d-cffd-4f28-8374-41fda9272270.png)

**Comment by Jonathan-LeRoux** [comment]
It looks like this isn't an issue with Microsoft but with your organization's administrator blocking Office apps from creating child processes. You'll have to inquire with your IT department whether they have a way to whitelist some apps or certain child processes.

**Comment by bernb** [comment]
> For example, if you save the .ppam file in `%appdata%\Microsoft\Addins` (i.e., `Users\user_name\Appdata\Roaming\Microsoft\Addins`), I think you should be able to use the add-in (unless your institution specifically prevents it).

This just resolved the issue for me, while changing Trust Center Settings didn't. If you could add this to the FAQ, I believe people will find this information much better, at least that's where I looked first. Maybe even add this as installation instructions?

Best regard and keep up your great work, I am super thankful you created that addin.

**Comment by Jonathan-LeRoux** [comment]
This is part of the [installation instructions on Windows](https://github.com/Jonathan-LeRoux/IguanaTex#windows-1) on the repo since Sep. 2022, but I realize that it indeed wasn't on the Readme for the current release 1.60.2 from August 2022.

I was about to push a new minor release, 1.60.3, so I took the opportunity to do it now.
I also realized that the LaTeXiT-metadata-Win.zip file that was attached to 1.60.2 was not the bug-fix one, despite that being mentioned in the release. It should be all good now.
